tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/apt/APTPropertiesPageTest.java

```diff
@@ -318,29 +341,21 @@ private void setProcessorCustomOutputFolder(String project, String customFolder)
 		e.save();
 	}
 	
-	
-	
-	private void checkBuildPath(String project, List<String> paths, String additionalPath){
+	private void checkBuildPath(String project, List<String> defaultPaths, List<String> additionalPaths){
 		PropertyDialog pd = openPropertiesProject(project);
 		new DefaultTreeItem("Java Build Path").select();
 		new DefaultTabItem("Source").activate();
 		List<TreeItem> items = new DefaultTree(1).getItems();
-		int expectedSize = 0;
-		if(additionalPath != null){
-			expectedSize = paths.size()+1;
-		} else {
-			expectedSize = paths.size();
-		}
-		String errMsg = "Additional path is: " + (additionalPath == null ? "" : additionalPath) + " , paths.size(): " + expectedSize + " and items.size(): " + items.size() ;
-		assertEquals(errMsg, expectedSize,items.size());
```

Comment:
@zcervink I suppose this check has been deleted by accident?

Reply:
@jkopriva thank you, I have corrected the accident. Tested on localhost, now testing on Jenkins (just to be sure everything is ok).

Reply:
@jkopriva tested on Jenkins, works fine:

<img width="1313" alt="jenkins" src="https://user-images.githubusercontent.com/43820055/52713782-825a4800-2f98-11e9-8cd2-ec61e840e9de.png">